### ISSUE TITLE
Support classes as a list as well as a vector.

### ISSUE THREAD

**Issue by milelo** [milelo/electric-hiccup#5]
How can a list be distinguished from a function call in a macro?

**Comment by milelo** [comment]
For practical purposes it is sufficient that is only support as return vales from functions, literal values must use vector. In this case it is easy to just vectorise lists returned by functions returning class values.

**Comment by milelo** [comment]
Addressed in 10c726882194061b6718c91a104a8552d41c4313

**Comment by milelo** [comment]
Released in:
```clojure
        milelo/electric-hiccup {:git/url "https://github.com/milelo/electric-hiccup"
                                :git/tag "v0.2.1-alpha"
                                :git/sha "78fee86"}
```
